.github/workflows/flake.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           # W503 and W504 are related as they conflict.  W503 is the preferred style and all code
           # should be using it now.
-          ignore: 'F841,W503,E402,F841'
```

Comment:
for completeness if you need to re-push can you remove one of the F841 entries from L39 as well, it's duplicated there.

E402 is import in the wrong place, I think we only have one of those so perhaps we could suppress it and remove from here as well?

Reply:
W503 is suppressed from the command line by https://github.com/daos-stack/daos/blob/master/.flake8

You need to either suppress W503 or W504 depending on your coding style, they conflict.